main.go

```diff
@@ -174,6 +177,12 @@ func readEnvVars() (map[string]string, error) {
 	}
 	envVars[sopEndpointEnvVarName] = val
 
+	val, found = os.LookupEnv(alertSMTPFromEnvVarName)
+	if !found {
+		return nil, fmt.Errorf("%s environment variable must be set", sopEndpointEnvVarName)
+	}
+	envVars[alertSMTPFromEnvVarName] = val
+
```

Comment:
As we see that a pattern emerges here, I would suggest a refactor as following:
```
	envVars := map[string]string{
		namespaceEnvVarName: "",
		addonNameEnvVarName: "",
		sopEndpointEnvVarName: "",
		alertSMTPFromEnvVarName: "",
	}
	for envVarName := range envVars {
		val, found := os.LookupEnv(envVarName)
		if !found {
			return nil, fmt.Errorf("%s environment variable must be set", envVarName)
		}
		envVars[envVarName] = val
	}
```

Also, the error message is using the wrong env name, `sopEndpointEnvVarName` and not `alertSMTPFromEnvVarName`